lib/VisualGenerator.js

```diff
@@ -220,7 +220,7 @@ class VisualGenerator {
         return new Promise((resolve, reject) => {
             let buildOptions = _.defaults(options, defaultOptions);
             let visualOptions = generateVisualOptions(visualName, buildOptions.apiVersion);
-            if (!visualOptions || !visualOptions.name) {
+            if (!visualOptions || !visualOptions.name || visualOptions.name.match(/^\d+/i)) {
```

Comment:
That's ok to prevent names that start at a number, but there's extra limitations such as ```do```, ```eval```, etc.. I suppose that it would be good to extend RegExp in order to get rid of other pitfalls. Please take a look at [this](https://stackoverflow.com/questions/1661197/what-characters-are-valid-for-javascript-variable-names) topic.